### PR TITLE
Feat : Added a feature for users to add feadback on github

### LINKED ISSUE #5563: Commons app feedback with reference to GitHub
### What is the user problem or growth opportunity you want to see solved?

From my point of view, it is an invaluable advantage that the Commons App allows you to leave feedback ( https://commons.wikimedia.org/wiki/Commons:Mobile_app/Feedback ) without having to be logged in to GitHub.

At the same time, I think it would be good if it were pointed out that feedback is better submitted via GitHub. Perhaps it would be enough to simply provide a link to GitHub! ( https://github.com/commons-app/apps-android-commons/issues )

### How do you know that this problem exists today? Why is this important?

Feedback is left on Commons, but processing is better if the feedback is stored directly on GitHub.

### Who will benefit from it?

all users

### Anything else you would like to add?

_No response_

### PR DESCRIPTION
**Pull request by neeldoshii**
## Description

* Fixes #5563 

## Approach
It gives the users an ability to place feedbacks (issues, new feature) directly on the github from the app.

## Testing Instructions
1. Open the More Menu from Bottom Navigation.
2. Bottom Sheet will be opened.
3. From the Bottom Sheet click on Send Feedback on Github
4. Browser will open (github issue section)

## Screenshots/ Working Video

https://github.com/commons-app/apps-android-commons/assets/60827173/f60d2287-db80-4cb0-b49b-ddb37eb0ea1d

## Tests performed 

Test performed on Google Pixel 4 (Emulator).

Feel free to place your opinion.

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/navtab/MoreBottomSheetFragment.java [hunk 1]
```diff
@@ -77,11 +77,20 @@ public View onCreateView(@NonNull final LayoutInflater inflater,
...
         binding.moreProfile.setOnClickListener(v -> onProfileClicked());
         binding.morePeerReview.setOnClickListener(v -> onPeerReviewClicked());
+        binding.moreFeedbackGithub.setOnClickListener(v -> onFeedbackGithubClicked());
 
         setUserName();
         return binding.getRoot();
     }
 
+    private void onFeedbackGithubClicked() {
+        final String url = "https://github.com/commons-app/apps-android-commons/issues";
```
Comment (shashankiitbhu) [inline app/src/main/java/fr/free/nrw/commons/navtab/MoreBottomSheetFragment.java:87]: can you make this a string constant?

**Review by shashankiitbhu — CHANGES_REQUESTED** [review]
Review by student

Before:

https://github.com/commons-app/apps-android-commons/assets/126143257/b780ae00-953f-4fef-a7cc-06bb40433a0c

After:

https://github.com/commons-app/apps-android-commons/assets/126143257/5b87f8b2-f62a-44d3-86dc-c9703d0c60e3

Mentioned Issue is Solved

File: app/src/main/java/fr/free/nrw/commons/navtab/MoreBottomSheetFragment.java (hunk 1 above)
Comment (neeldoshii) [reply app/src/main/java/fr/free/nrw/commons/navtab/MoreBottomSheetFragment.java]: fixed. Thanks for the feedback.

**Review by shashankiitbhu — APPROVED** [review]

**Review by nicolas-raoul — APPROVED** [review]
Thanks! :-)

**Comment by ShashwatKedia** [comment]
@nicolas-raoul Sorry to ping after the PR has been merged, but this PR has two test cases failing [here](https://github.com/commons-app/apps-android-commons/actions/runs/8505891388/job/23295149636):

`fr.free.nrw.commons.description.DescriptionEditActivityUnitTest > testUpdateDescription FAILED
    java.lang.reflect.InvocationTargetException at DescriptionEditActivityUnitTest.kt:120
        Caused by: fr.free.nrw.commons.auth.csrf.InvalidLoginTokenException at DescriptionEditActivityUnitTest.kt:120`

`fr.free.nrw.commons.description.DescriptionEditActivityUnitTest > testOnSubmitButtonClicked FAILED
    java.lang.reflect.InvocationTargetException at DescriptionEditActivityUnitTest.kt:132
        Caused by: fr.free.nrw.commons.auth.csrf.InvalidLoginTokenException at DescriptionEditActivityUnitTest.kt:132`
        
This is causing the tests to fail on all other PRs. Also, while I was testing the main branch, I was surprised to see that this PR was merged, when there was still some discussion going on [here](https://github.com/commons-app/apps-android-commons/issues/5563#issuecomment-1974803819) about the best possible UI for feedback on GitHub. And as mentioned there, even you didn't approve of the concept of having 2 separate menu items.

**Comment by nicolas-raoul** [comment]
Thanks @ShashwatKedia for pointing this out!
Actually, even after I revert 54312fc2ac1bc88b03539c7627e6267a49f79b9d these tests fail.
Maybe an earlier commit? git bisect would help here.

I agree two menu items is not ideal, but I felt that it was still better than nothing.

**Comment by nicolas-raoul** [comment]
git bisect says 3d1efecb55c931ffe55137bff7e69671ef5cada8 is the first commit where unit tests start failing.

**Comment by ShashwatKedia** [comment]
Oh, thanks and sorry, @nicolas-raoul, that you had to perform git bisect for this. I'll look into this and try to perform the actions required by PR without failing the tests

**Comment by ShashwatKedia** [comment]
> I agree two menu items is not ideal, but I felt that it was still better than nothing.

Um, what do you think about the solution proposed [here](https://github.com/commons-app/apps-android-commons/issues/5563#issuecomment-1975746623) ?

**Pull request merged**